portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,36 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final ArrayList<String> COMMENTS = new ArrayList<>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Kevin!</h1>");
+    initializeComments();
```

Comment:
this is actually a bit weird to see a constants getting initialized in the get function.
my suggestion is to initialize it in the init function.

Reply:
Not too sure if any such init function will be called before doGet. My understanding is that once the GET request is submitted, only the doGet function will be called and thus I have to handle all the code there. Maybe there's an annotation which I can use for this. Anyway, the initializeComments() method is removed in the datastore branch, which is the final code that I want to merge.

Reply:
are you able to use this?
https://stackoverflow.com/a/5140494

Reply:
Done. Thanks for the suggestion :)